light-client/src/components/io.rs

```diff
@@ -147,7 +153,11 @@ mod prod {
             }
         }
 
-        fn fetch_validator_set(&self, height: AtHeight) -> Result<TMValidatorSet, IoError> {
+        fn fetch_validator_set(
+            &self,
+            height: AtHeight,
+            proposer_address: Option<TMAccountId>,
```

Comment:
TMAccountId may not be the best name for validator ID. accounts usually refer to users accounts who have wallets & transact within a network

Reply:
That's the name of the type currently used for the address field in `validator::Info`: https://github.com/informalsystems/tendermint-rs/blob/c5fe2a88c80d9336bd14a9bcbf575e7a62f3a93b/tendermint/src/validator.rs#L145-L146

Perhaps we need a new type for this?

Reply:
actually, validators also receive a reward. please ignore